File: Plugins/Aspose_Tasks_Java_for_Ruby/lib/asposetasksjava/Currencies/currencydigits.rb
module Asposetasksjava
  module CurrencyDigits
    def initialize()
    	# Getting Currency Digits
    	get_currency_digits()

    	# Getting Currency Digits
    	set_currency_digits()
    end

    def get_currency_digits()
        data_dir = File.dirname(File.dirname(File.dirname(File.dirname(__FILE__)))) + '/data/'
                
        # Instantiate project object
        project = Rjb::import('com.aspose.tasks.Project').new(data_dir + 'test_tasks.mpp')

        puts "Currency Digits: " + project.get(Rjb::import('com.aspose.tasks.Prj').CURRENCY_DIGITS).to_string
    end

    def set_currency_digits()
        data_dir = File.dirname(File.dirname(File.dirname(File.dirname(__FILE__)))) + '/data/'
                
        # Instantiate project object
        project = Rjb::import('com.aspose.tasks.Project').new(data_dir + 'test_tasks.mpp')

        project.set(Rjb::import('com.aspose.tasks.Prj').CURRENCY_DIGITS, 2)

        project.save(data_dir + "ProjectCurrDigits.mpp", Rjb::import('com.aspose.tasks.SaveFileFormat').MPP)

        puts "Set currency digits."
    end
  end
end

File: Plugins/Aspose_Tasks_Java_for_PHP/src/aspose/tasks/WorkingWithCurrencies/CurrencyDigits.php
<?php
namespace Aspose\Tasks\WorkingWithCurrencies;

use com\aspose\tasks\Project as Project;
use com\aspose\tasks\Prj as Prj;
use com\aspose\tasks\SaveFileFormat as SaveFileFormat;
class CurrencyDigits{

    public static function run($dataDir=null){

        # Getting Currency Digits
        CurrencyDigits::get_currency_digits($dataDir);

    	# Getting Currency Digits
    	CurrencyDigits::set_currency_digits($dataDir);
    }

    public static function get_currency_digits($dataDir=null){

        # Instantiate project object
        $project = new Project($dataDir . 'test_tasks.mpp');

        $prj=new Prj();
        print "Currency Digits: " . (string)$project->get($prj->CURRENCY_DIGITS).PHP_EOL;//.to_string
    }

    public static function set_currency_digits($dataDir=null){

        # Instantiate project object
        $project = new Project($dataDir . 'test_tasks.mpp');

        $prj=new Prj();
        $project->set($prj->CURRENCY_DIGITS, 2);

        $saveFileFormat=new SaveFileFormat();
        $project->save($dataDir . "ProjectCurrDigits.mpp", $saveFileFormat->MPP);
        print "Set currency digits.".PHP_EOL;
    }

}
?>